### PR TITLE
Schedule account booking

### PR DESCRIPTION
**Pull request by valentinbiz**

### CODE REVIEW

**Comment by m-thapa** [comment]
Pull away

**Review by m-thapa — APPROVED** [review]

File: components/Calendar.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+// import React from "react";
```
Comment (iy2k22) [inline components/Calendar.jsx:1]: Please delete this if you're not using it.

File: components/UserContext.jsx [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import React, { useState, createContext } from 'react';
```
Comment (iy2k22) [inline components/UserContext.jsx:1]: Please could you place this in the contexts folder rather than the components folder?

File: index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+// import { StyleSheet, View, Text} from "react-native";
```
Comment (iy2k22) [inline index.js:1]: Please delete these lines if you're not using them.

File: pages/Account.jsx [hunk 4]
```diff
@@ -1,60 +1,108 @@
...
 } from "react-native";
+import { AntDesign } from "@expo/vector-icons";
+import * as ImagePicker from "expo-image-picker";
+import FormInput from "../components/FormInput";
+import { NavigationContainer, useNavigation } from "@react-navigation/native";
+   
 
-function Account() {
+export default function Account({navigation}) {
+  const navigated = useNavigation();
```
Comment (iy2k22) [inline pages/Account.jsx:24]: It seems as though you've forgotten to delete references to useNavigation. Please delete both the import and the navigated variable if you're not using them.

**Pull request merged**